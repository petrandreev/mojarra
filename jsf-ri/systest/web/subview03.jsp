<!--
 Copyright 2004 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->
<%@ page contentType="text/html" language="java" %>
<%@ taglib prefix="c" uri="http://java.sun.com/jstl/core" %>
<%@ taglib prefix="f" uri="http://java.sun.com/jsf/core" %>
<%@ taglib prefix="h" uri="http://java.sun.com/jsf/html" %>

<f:view>
<html>
<head>
<title>subview03</title>
</head>
<body>
<h:outputText value="Begin test <c:include> with subview tag in included page"/>
<jsp:include page="foo01.jsp"/>
<h:outputText value="subview03"/>
<jsp:include page="bar01.jsp"/>
<h:outputText value="End test <c:include> with subview tag in included page"/>
</body>
</html>
</f:view>
