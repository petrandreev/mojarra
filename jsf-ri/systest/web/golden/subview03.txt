<!--
 Copyright 2004 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->






<html>
<head>
<title>subview03</title>
</head>
<body>
<p>
Begin test &lt;c:include&gt; with subview tag in included page</p>

<p><!--
 Copyright 2004 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->





foo01
</p>

<p>subview03</p>

<p><!--
 Copyright 2004 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->





bar01
</p>

<p>End test &lt;c:include&gt; with subview tag in included page</p>
</body>
</html>

